webapp/src/Controller/Jury/SubmissionController.php

```diff
@@ -1129,4 +1129,13 @@ protected function processClaim($judging, Request $request, ?string &$claimWarni
 
         return null;
     }
+
+    /**
+     * @Route("/{submitId<\d+>}/create-tasks", name="jury_submission_create_tasks")
+     */
+    public function createJudgeTasks(string $submitId): RedirectResponse {
```

Comment:
This promises to return a `RedirectResponse` but doesn't do so.

Reply:
I assume `return $this->redirectToRoute` would return that response, how did you find this mistake?